extensions/team-time/src/types.ts

```diff
@@ -0,0 +1,39 @@
+export type TimeEntry = {
+  id: string;
+  city: string;
+  country: string;
+  isoCode: string;
+  timeZone: string;
+  state?: string;
+};
+
+// Type definitions for the JSON structure
+export type TimeZoneData = {
+  zone: string;
+  cities: string[];
+};
+
+export type StateData = {
+  state: string;
+  timeZones: TimeZoneData[];
+};
+
+export type CountryData = {
+  country: string;
+  isoCode: string;
+  states?: StateData[];
+  timeZones?: TimeZoneData[];
+};
+
+export type TimeGroup = {
+  time: string;
+  signature: string;
+  cities: string[];
+  combinedFlags: string;
+};
+
+export type FormatTimePreference = {
+  timeFormat: "24h" | "12h";
+};
+
```

Comment:
```suggestion
```

You don't need to manually set preference types as this is autogenerated in `raycast-env.d.ts`